release/nightly_tests/dask_on_ray/1tb_sort_compute.yaml

```diff
@@ -10,13 +10,13 @@ aws:
 
 head_node_type:
     name: head_node
-    instance_type: i3.8xlarge
+    instance_type: m5.8xlarge
```

Comment:
why not change to `m6i`? which is cheaper and faster?

Reply:
wow cheaper and faster exists?